src/main/scala/com.snowplowanalytics.snowplow.micro/ConfigHelper.scala

```diff
@@ -44,10 +44,11 @@ private[micro] object ConfigHelper {
   implicit val sinkConfigHint = new FieldCoproductHint[SinkConfig]("enabled")
 
   /** Parse the command line arguments and the configuration files. */
-  def parseConfig(args: Array[String]): (CollectorConfig, Resolver[Id], IgluCirceClient[Id], Config) = {
+  def parseConfig(args: Array[String]): (CollectorConfig, Resolver[Id], IgluCirceClient[Id], Config, Boolean) = {
```

Comment:
I know this is getting messy. We should return a case class instead. But maybe that can be left for a later refactor...